packages/frontend/src/utils/apollo-client.ts

```diff
@@ -11,7 +11,7 @@ const httpLinkRP = new HttpLink({
 })
 
 const httpLinkRPSqueeth = new HttpLink({
-  uri: 'https://api.thegraph.com/subgraphs/name/opynfinance/squeeth-ropsten',
+  uri: 'https://api.thegraph.com/subgraphs/name/yichiehliu/squeeth',
```

Comment:
Can we put this deployment under opyn? Or were there issues w doing that?

Reply:
Yh can do it after #125 merged, now only mine has the updated action types